src/Symfony/Bridge/Doctrine/Form/ChoiceList/ORMQueryBuilderLoader.php

```diff
@@ -37,42 +35,10 @@ class ORMQueryBuilderLoader implements EntityLoaderInterface
     /**
      * Construct an ORM Query Builder Loader.
      *
-     * @param QueryBuilder|\Closure $queryBuilder The query builder or a closure
-     *                                            for creating the query builder.
-     *                                            Passing a closure is
-     *                                            deprecated and will not be
-     *                                            supported anymore as of
-     *                                            Symfony 3.0.
-     * @param ObjectManager         $manager      Deprecated.
-     * @param string                $class        Deprecated.
-     *
-     * @throws UnexpectedTypeException
+     * @param QueryBuilder $queryBuilder The query builder for creating the query builder.
      */
-    public function __construct($queryBuilder, $manager = null, $class = null)
+    public function __construct(QueryBuilder $queryBuilder)
```

Comment:
The instantiation in the EntityType class should also be changed to pass only the necessary argument.